com.github.marinm.songrec.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "com.github.marinm.songrec",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "songrec",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=host:ro",
+        "--filesystem=/tmp:ro",
+        "--filesystem=xdg-config",
+        "--filesystem=xdg-cache",
```

Comment:
```suggestion
```
I don't think you need access to those ?

Reply:
Sorry, I actually store things in `~/.local/share/SongRec`, fixed in commit 74c85ee